docs/notebooks/analysis/swept_langmuir/find_floating_potential.ipynb

```diff
@@ -0,0 +1,390 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Swept Langmuir Analysis: Floating Potential\n",
```

Comment:
I think it would be helpful to start off the notebook with some higher level context, like what a floating potential is, what a swept Langmuir trace is (and is for), and why we want to go about finding it.  It needn't be super detailed, but just enough for someone to have an idea of the big picture.

Reply:
This might also be a good place to define V_f as the floating potential.

Reply:
Yeah, I've been going back and forth about this.  I'm debating about organizing the notebooks in one of two ways:

1. Have a notebook for each swept Langmuir function that both describes the function's behavior and the physics behind the parameter being calculated.
2. Have a note book for each swept Langmuir functionality that only describes the functions behavior.  Then have a separate notebook that walks through the whole swept Langmuir analysis from beginning to end that focuses more on the swept Langmuir physics and analysis workflow.

I haven't decided which I prefer, but I'm leaning more towards the later.  I won't really know until I get more functionality incorporated.  For expidient reasons, I kept this notebook based more on functionality and plan to come back after I decide on which of the above options I want to go with.  So, I'd rather keep notebook's scope as is and come back in a followup PR.

Reply:
Sounds good!  For the notebooks for each swept Langmuir function, I think it'd still be helpful to have a sentence or two to connect the notebook to the big picture, but that'll probably be easier to do once the notebook that goes through the whole swept Langmuir analysis is done.

Reply:
Or, leave documentation of the low-level functionality to the documentation contained within the functions themselves; definitely keep the high level notebook for the whole walkthrough process; and have short notebooks that describe the physics of why one needs to fit  various portions of the curve in detail with examples of how this is accomplished with the various functions. I think this translates to 'notebooks for physics' and 'no notebooks dedicated to functions'